src/__tests__/jsonApiLink.ts

```diff
@@ -2036,116 +2054,124 @@ describe('Mutation', () => {
         );
       }
     });
-  });
 
-  it.skip('returns an error on zero Content-Length but status > 300', async () => {
-    expect.assertions(1);
+    it('returns an error on zero Content-Length but status > 300', async () => {
+      expect.assertions(1);
 
-    const link = new JsonApiLink({ uri: '/api' });
+      const link = new JsonApiLink({ uri: '/api' });
 
-    const post = { id: '1', title: 'Love apollo' };
-    fetchMock.post('/api/posts', {
-      headers: { 'Content-Length': 0 },
-      status: 500,
-      body: post,
-    });
+      const post = { id: '1', title: 'Love apollo' };
+      fetchMock.post('/api/posts', {
+        headers: { 'Content-Length': 0 },
+        status: 500,
+        body: post,
+      });
 
-    const createPostMutation = gql`
-      fragment PublishablePostInput on REST {
-        title: String
-      }
+      const createPostMutation = gql`
+        fragment PublishablePostInput on REST {
+          title: String
+        }
```

Comment:
This fragment structure isn't JSON API either. I think we can actually remove this here.

Reply:
Ok, I'll remove the fragment.